var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -181,7 +181,7 @@ def patch(self):
 
     @classmethod
     def determine_version(cls, exe):
-        perl = spack.util.executable.Executable(exe)
+        perl = llnl.util.executable.Executable(exe)
```

Comment:
`Executable` is already imported right? Shouldn't we just use:
```suggestion
        perl = Executable(exe)
```

Reply:
fixed